### ISSUE TITLE
Can't find Catmandu::Fix::lido_recordwrap

### ISSUE THREAD

**Issue by netsensei** [PACKED-vzw/Catmandu-Fix-LIDO#2]
Test case: [the collection data of MoMa NY](https://github.com/MuseumofModernArt/collection).

Imported the data in a MongoDB instance. I'm now trying to convert them to barebones LIDO using the fixes. so:

`carton exec catmandu export MongoDB --database_name moma to LIDO --fix fix.txt` with moma being the db containing the JSON representation found at the git repo above.

I get this error:

```
Oops! Tried to execute the fix 'lido_recordwrap' but can't find Catmandu::Fix::lido_recordwrap on your system.
Source:

    add_field('WorkType', 'Painting')
    add_field('recordSource', 'Museum of Modern Art NY')
    add_field('recordType', 'item')
    lido_recid(ObjectID, 'test')
    lido_title(Title, Title)
    lido_worktype(WorkType)
    lido_recordwrap(ObjectID, recordType, recordType, recordSource)
```

The above 8 or so lines are my fix stored in fix.txt. It's just a test.

Looked at the lido_recordwrap.PM file but can't see anything out of the ordinary at first glance.

**Comment by pieterdp** [comment]
How did you install Catmandu-Fix-LIDO? If it was/is not installed via CPAN, is it in your `PERL5LIB` or `@INC`?

**Comment by netsensei** [comment]
Using [carton + cpan](https://gist.github.com/netsensei/e019fdbc413e4525e2120359e140526d) to get it up and running. 

Seems, the issue was a stray argument in a `use` statement. I removed it, and it started to work magically for me.  (Still figuring out Perl right now)

![](http://www.reactiongifs.com/r/mgc.gif)

Submitted a pull request

**Comment by pieterdp** [comment]
Closed by  dcce5b3.
